nucypher/cli/deploy.py

```diff
@@ -72,8 +72,9 @@ def deploy():
 @deploy.command(name='download-registry')
 @click.option('--config-root', help="Custom configuration directory", type=click.Path())
 @click.option('--registry-outfile', help="Output path for contract registry file", type=click.Path(file_okay=True))
+@click.option('--network', help="", type=click.Choice(['goerli']), default='goerli')  # TODO: #1496
```

Comment:
Is it possible to use `CanonicalRegistrySource.networks` values for the choice type, instead of needing to manually add to the choice when mainnet rolls around?

Reply:
Ah, that's a great idea! Although strictly better to current approach, I don't know if when the moment comes we'd end up extracting the networks list to some other class. But yeah, in the meantime, let's do this.